### PR TITLE
Created the VO for Url

### PR DESCRIPTION
**Pull request by sdecandelario**
Created the VO for an Url, added a new folder called Exceptions to better maintenance of all kinds of exceptions and to make it easy to manage new exceptions.

Added two tests for the new VO.

### CODE REVIEW

File: src/Internet/Url.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public function __construct(string $url)
+    {
+        $this->validate($url);
+        parent::__construct($url);
+    }
+
+    private function validate(string $url)
+    {
+        if (filter_var($url, FILTER_VALIDATE_URL) === false) {
+            throw InvalidUrlException::create($url);
```
Comment (jgimeno) [inline src/Internet/Url.php]: Launch InvalidArgumentException, to be consistent we don't use custom exceptions.

File: src/Exceptions/InvalidUrlException.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?php
```
Comment (jgimeno) [inline src/Exceptions/InvalidUrlException.php]: This class is not needed

File: src/Internet/Url.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+namespace ValueObjects\Internet;
+
+use ValueObjects\Exceptions\InvalidUrlException;
+use ValueObjects\ValueObject;
+
+/**
+ * Url
+ *
+ * @author Sergio de Candelario <sergio.decandelario@gmail.com>
```
Comment (jgimeno) [inline src/Internet/Url.php]: We don't put author in the Phpdocs, take a look at other examples in the code. :D

File: src/Internet/Url.php (hunk 1 above)
Comment (sdecandelario) [reply src/Internet/Url.php]: Okey, but i think it's better to launch custom exceptions, to can make some rules, but i cahnge it

**Pull request merged**